### PR TITLE
Add anubis

### PR DESCRIPTION
**Pull request by wykwit**
See: #3791

### CODE REVIEW

File: packages/anubis/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+depends=('python' 'python-python-nmap' 'python-shodan' 'python-docopt'
+         'python-requests' 'python-censys' 'python-dnspython')
+makedepends=('git' 'python-build' 'python-pip')
+source=("git+https://github.com/jonluca/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
```
Comment (noraj) [inline packages/anubis/PKGBUILD:22]: There are git tags https://github.com/jonluca/Anubis/tags so you should use https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-generic-git#L24 instead

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/anubis/PKGBUILD (hunk 1 above)
Comment (wykwit) [reply packages/anubis/PKGBUILD:22]: Isn't this exactly L24 of that template?

**Review by noraj — APPROVED** [review]
LGTM

**Comment by noptrix** [comment]
@wykwit thanks for this

**Pull request merged**